style.css

```diff
@@ -514,9 +514,7 @@ button:disabled {
     align-items: center;
     width: 142px;
     height: 48px;
-    margin-top: 10px;
-    margin-bottom: 50px;
-    margin-left: 330px;
+    margin: 2rem auto 0 9rem;
   }
```

Comment:
- [ ] Please know that the button is still positioned in the wrong way, this is how it looks:

![image](https://user-images.githubusercontent.com/20567503/179372577-0af89ed8-dd5f-4295-8de8-61719dd7df57.png)

Kindly make sure that it's positioned in the center so it looks like the Figma design.

Please note that you're setting the `margin-right` to `auto` in the above code, which is the reason the button is leaning towards the right, to solve the issue and make it centered just use the container of the button which has an id of `form-button`, make that container a `flexbox` and center the button using it without the need to set any margin in the button itself.